database_admin/migrations/068_package_latest_fix.up.sql

```diff
@@ -0,0 +1,32 @@
+DROP FUNCTION IF EXISTS refresh_latest_packages_view;
+DROP INDEX IF EXISTS package_latest_cache_pkey;
+DROP MATERIALIZED VIEW IF EXISTS package_latest_cache;
+
+
+CREATE MATERIALIZED VIEW IF NOT EXISTS package_latest_cache
+AS
+SELECT DISTINCT ON (p.name_id) p.name_id, p.id as package_id, sum.value as summary
+FROM package p
+         INNER JOIN strings sum on p.summary_hash = sum.id
+         LEFT JOIN advisory_metadata am on p.advisory_id = am.id
+ORDER BY p.name_id, am.public_date DESC NULLS LAST, sum.value DESC NULLS LAST;
```

Comment:
I think we can simplify that:
~~~
ORDER BY p.name_id, am.public_date, sum.value DESC NULLS LAST;
~~~

Reply:
Ok, it seems we can not. Ordering is specified for each column.